### ISSUE TITLE
[WHM] Afflatus Feature doesn't replace Medica III with Afflatus Rapture

### ISSUE THREAD

**Issue by ganderhero** [MKhayle/XIVComboExpanded#254]
![Screenshot_26](https://github.com/MKhayle/XIVComboExpanded/assets/119761174/c934fcec-ae9a-4f82-be09-762fee285832)
Title, it replaces Cure II with Afflatus Solace but it doesn't replace Medica III with Afflatus Rapture

**Comment by MKhayle** [comment]
I'll add compatibility with the new Medica III (that's a new spell right?) for that combo
Also, can you confirm whether the Glare4 feature is working correctly?

**Comment by MKhayle** [comment]
Oh, wait, Medica III is a Medica II update. 

This combo is supposed to replace Medica (the lvl 10 spell) only, since a Lily's Afflatus is a straight upgrade for Medica, while Medica II/III have a different goal (lower instant healing potency but with higher healing over time), that's why it doesn't replace Medica III (and it won't).

**Comment by ganderhero** [comment]
@MKhayle Yes the Glare IV feature works perfectly when applying Presence of mind, is there any way to add in the option for Afflatus Rapture to also replace Medica II/III then?

**Comment by MKhayle** [comment]
I'll consider it, remind me if I forgot by the end of July!

**Comment by MKhayle** [comment]
should be added but untested for medica III

**Comment by MKhayle** [comment]
Any confirmation of whether it's working or not?

**Comment by ganderhero** [comment]
Yes, it works perfectly well, it's replacing Medica II/III correctly when a lily is avaliable
